app.js

```diff
@@ -15,6 +17,10 @@ app.use(express.static("public"));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
+if (auth_config != undefined) {
```

Comment:
It's mostly a style preference but I think just checking for truthiness is sufficient here unless you specifically care about null vs undefined and I think we don't here?

`if (auth_config)`